### PR TITLE
fix breaking examples

### LINKED ISSUE #4997: Spring Boot example does not work
### What happened?

On Windows I ran `kubectl kustomize` in the springboot/overlays/production example which resulted in the following output:

> error: accumulating resources: accumulation err='accumulating resources from '../../base': 'C:\Code\github\kustomize\examples\springboot\base' must resolve to a file': couldn't make target for path 'C:\Code\github\kustomize\examples\springboot\base': unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory 'C:\Code\github\kustomize\examples\springboot\base'

### What did you expect to happen?

Something different than the error output.

### How can we reproduce it (as minimally and precisely as possible)?

Check the spring-boot example contained in this repo

### Expected output

_No response_

### Actual output

_No response_

### Kustomize version

4.5.7

### Operating system

Windows

### PR DESCRIPTION
**Pull request by koba1t**
fix breaking examples in `./examples` dir.
fix #4997

### CODE REVIEW

File: examples/loadHttp/kustomization.yaml [hunk 1]
```diff
@@ -4,12 +4,11 @@ resources:
 - https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/examples/wordpress/wordpress/deployment.yaml
 - https://github.com/knative/serving/releases/download/v0.12.0/serving.yaml # redirects to s3
 patches:
```
Comment (koba1t) [inline examples/loadHttp/kustomization.yaml:6]: removed legacy patch syntax is used.

File: examples/springboot/base/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+resources:
```
Comment (koba1t) [inline examples/springboot/base/kustomization.yaml:1]: error caused by `kustomization.yaml` file is missing.

**Comment by KnVerey** [comment]
Are the fixes taken from the tutorial snippets? Should we update them there instead? I don't feel strongly about it, but the tutorial is telling the user to create these files, isn't it? https://github.com/kubernetes-sigs/kustomize/tree/master/examples/springboot

**Comment by koba1t** [comment]
@KnVerey 

>  the tutorial is telling the user to create these files

That is right.
So, Do you think better for removing non-snippet files?
Creating a few files is required using commands on the tutorials step, but the files included above dirs. (ex. `patch.yaml`) https://github.com/kubernetes-sigs/kustomize/blob/0fd385d7197026b92272dc8688ec66593be91fba/examples/springboot/README.md#L107-L119

File: examples/loadHttp/kustomization.yaml (hunk 1 above)
Comment (cailynse) [reply examples/loadHttp/kustomization.yaml:6]: This is used in other examples - is it worth updating those as well?

File: examples/springboot/base/kustomization.yaml (hunk 2 above)
Comment (cailynse) [reply examples/springboot/base/kustomization.yaml:1]: Does this mean the example in the issue never worked?

File: examples/loadHttp/kustomization.yaml (hunk 1 above)
Comment (koba1t) [reply examples/loadHttp/kustomization.yaml:6]: @cailynse  

I think I searched and fixed other examples where the removed legacy patch was used.
Could you find other places for it was used?

Comment (cailynse) [reply examples/loadHttp/kustomization.yaml:6]: Sorry! I misunderstood, I was thinking `patchesStrategicMerge` which is being used in many places

**Review by tonglil — APPROVED** [review]
Please fix the examples, thanks!

**Pull request merged**